v2/yarpcgrpc/outbound.go

```diff
@@ -194,7 +195,7 @@ func invokeErrorToYARPCError(err error, responseMD metadata.MD) error {
 	status, ok := status.FromError(err)
 	// if not a yarpc error or grpc error, just return a wrapped error
 	if !ok {
-		return yarpcerror.FromError(err)
+		return yarpcerror.WrapError(err)
```

Comment:
As with @AlexAPB’s suggestion of removing the Error stutter from yarpcerror.Extract, I think it makes sense to yarpcerror.Wrap(err). Take or leave.

I know this makes me a terrible person, but among functional programmers, this function would probably be called yarpcerror.Lift(err) because it lifts the error type to the yarpc error level of abstraction, but does nothing if it is already at that level.